cpp/include/cudf/io/orc.hpp

```diff
@@ -57,10 +57,10 @@ class orc_reader_options {
 
   // List of individual stripes to read (ignored if empty)
   std::vector<std::vector<size_type>> _stripes;
-  // Rows to skip from the start; ORC stores the number of rows as uint64_t
-  uint64_t _skip_rows = 0;
+  // Rows to skip from the start
+  int64_t _skip_rows = 0;
```

Comment:
why this change?

Reply:
Imagine that we have 100 billion rows. We want to skip 50B rows, and want to read 10B rows. We can do that only with `skip_rows` and `num_rows` are 64 bit.